src/index.js

```diff
@@ -176,37 +182,49 @@ function addMetadata({ types: t }) {
       CallExpression(babelPath, state) {
         if (!state.opts.shouldLoadFile) return;
 
-        // If we're at a top-level call expression, then we reset the beforeEachModified state to false, and let Babel
-        // move on to the next call expression.
-        if (babelPath.scope.block.type === 'Program') {
-          state.opts.beforeEachModified = false;
-          return;
+        // If this call expression is a top-level module, store string arg, else skip the nested module entirely
+        if (babelPath.get('callee').isIdentifier({ name: 'module' })) {
+          if (babelPath.parentPath.parent.type === 'Program') {
```

Comment:
I realize this makes it a bit more terse, but I find it makes this harder to read. I now have to do the conditional in my head rather than just reading the cascade. 

In most cases I'd agree with you, in this case I don't.